settings/nitrofiles/languages/english.ini

```diff
@@ -53,17 +53,17 @@ SLOW=Slow
 
 DESCRIPTION_FRAMERATE=Changes how many frames per second are displayed.
 DESCRIPTION_DSCLASSICMENU=The menu that is shown by pressing SELECT in the DSi theme, can be shown before the ROM select menu.
-DESCRIPTION_SELECTBUTTONOPTION=Choose a menu to show, when pressing SELECT in the DSi or SEGA Saturn theme.
+DESCRIPTION_SELECTBUTTONOPTION=Choose a menu to show when pressing SELECT in the DSi or SEGA Saturn theme.
```

Comment:
Select which menu to show when pressing SELECT in the Nintendo DSi, SEGA Saturn and Homebrew launcher themes.